pkg/specgen/generate/pause_image.go

```diff
@@ -4,18 +4,14 @@ package generate
 
 import (
 	"context"
-	"fmt"
-	"os"
 
-	buildahDefine "github.com/containers/buildah/define"
 	"github.com/containers/common/pkg/config"
 	"github.com/containers/podman/v5/libpod"
-	"github.com/containers/podman/v5/libpod/define"
 )
 
-// PullOrBuildInfraImage pulls down the specified image or the one set in
+// PullInfraImage pulls down the specified image or the one set in
 // containers.conf.  If none is set, it builds a local pause image.
-func PullOrBuildInfraImage(rt *libpod.Runtime, imageName string) (string, error) {
+func PullInfraImage(rt *libpod.Runtime, imageName string) (string, error) {
```

Comment:
comment needs updating above. Also it should document that one can pass an empty name and the function may return an empty name in which case the default catatonit rootfs logic in libpof must be used.